collector/netstat.go

```diff
@@ -35,10 +36,19 @@ func NewNetStatCollector() (Collector, error) {
 }
 
 func (c *netStatCollector) Update(ch chan<- prometheus.Metric) (err error) {
-	netStats, err := getNetStats()
+	netStats, err := getNetStats(procNetStat)
 	if err != nil {
 		return fmt.Errorf("couldn't get netstats: %s", err)
 	}
+	snmpStats, err := getNetStats(procSnmpStat)
+	if err != nil {
+		return fmt.Errorf("couldn't get netstats: %s", err)
```

Comment:
Yes, thought of that, but then the problem comes up at line 68 as well. Maybe I'll just change line 68 then to 'couldn't distinguish stats'. Good call though, I'll change it.